rbc/libfuncs.py

```diff
@@ -56,6 +56,7 @@ class HeavyDB(Library):
     allocate_varlen_buffer set_output_row_size
     TableFunctionManager_error_message TableFunctionManager_set_output_row_size
     TableFunctionManager_set_output_array_values_total_number
+    TableFunctionManager_set_output_item_values_total_number
```

Comment:
A note: this list of function names depends on the heavydb version. I wonder if this fact is practically relevant.